deploy/tasks/assets.php

```diff
@@ -0,0 +1,57 @@
+<?php
+
+namespace Deployer;
+
+// parameters
+set('asset_resource_paths', [
+    'src/AppBundle/Resources/es6',
+    'src/AppBundle/Resources/sass'
+]);
+set('asset_build_base', 'web');
+set('asset_build_paths', [
+    'assets',
+]);
+set('asset_exclude_paths', [
+   'assets/app/build_dev'
+]);
+set('asset_install_command', 'yarn install');
+set('asset_build_command', 'yarn encore production');
```

Comment:
We should use `yarn build:prod` here, to make it possible to change the command through `package.json`. Besides, that's what we use anyway when building assets now.

Reply:
There is also `yarn prod`, that is why I was not sure which to use. Maybe we should remove duplicates from the config to avoid confusion?

Reply:
Fine with me. We can run the poll on Slack to see which of those is more frequently used.

Reply:
Question asked on Slack, will adjust the PR later based on the outcome :)

Reply:
![image](https://user-images.githubusercontent.com/5854428/108832208-7cf25800-75cb-11eb-9b90-60b6e9208141.png)

Changed in 8981bdd6